sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -40,9 +40,17 @@
                 " /></v-col
           ></v-row>
         </content-group>
-        <v-divider></v-divider>
+        <v-divider
+          v-if="
+            restrictionData.isActive &&
```

Comment:
this condition is repeating `restrictionData.isActive &&  restrictionData.restrictionCategory !== 'Federal'` and hence it could be a computed property